Code/Framework/AzQtComponents/AzQtComponents/Components/Widgets/TableView.qss

```diff
@@ -30,7 +30,6 @@ QTreeView
 
 AzQtComponents--TableView::item,
 QTableView::item,
-QListView::item,
```

Comment:
Is it possible to add a line in Editor.qss to make a special case for this panel instead of altering the whole TableView's stylesheet? I'm afraid this change may have unforeseen consequences and would rather special-case this list view.

Reply:
Looks like it should work like that... maybe you can also try

```
CNewLevelDialog QListWidget::item
```